packages/account/src/Components/account-limits/account-limits-table-cell.tsx

```diff
@@ -1,9 +1,19 @@
-import PropTypes from 'prop-types';
 import * as React from 'react';
 import classNames from 'classnames';
 import { Text } from '@deriv/components';
 
-const AccountLimitsTableCell = ({ align, children, is_hint, renderExtraInfo }) => {
+type TAccountLimitsTableCell = {
+    align?: 'right' | 'left';
+    is_hint?: boolean;
+    renderExtraInfo?: () => React.ReactElement;
+};
+
+const AccountLimitsTableCell = ({
+    align,
+    children,
+    is_hint,
+    renderExtraInfo,
+}: React.PropsWithChildren<Partial<TAccountLimitsTableCell>>) => {
```

Comment:
when we use partial, that means all propr are optional. we don't need to use ? in types

Reply:
Owhhh okok will edit based on your comment 👌